src/games/ecards.py

```diff
@@ -84,6 +84,18 @@ async def start(self):
         }
 
         # run a round for each player
+        return await self.execute_rotation(prompt_deck, answer_deck, hands)
+
+    @EGameFactory.execute_rounds()
```

Comment:
To save me having to look at what the default are, can you explicitly put them in here (even if they are just the default values)?